web/src/features/fba/pages/FireBehaviourAdvisoryPage.tsx

```diff
@@ -152,17 +154,55 @@ const FireBehaviourAdvisoryPage: React.FunctionComponent = () => {
     document.title = ASA_DOC_TITLE
   }, [])
 
+  const formControlRef = useRef<HTMLDivElement>(null)
+  const mapRef = useRef<HTMLDivElement>(null)
+  const navRef = useRef<HTMLDivElement>(null)
+  const sidePanelRef = useRef<HTMLDivElement>(null)
+  const [formControlHeight, setFormControlHeight] = useState<number>(0)
+  const [navRefHeight, setNavRefHeight] = useState<number>(0)
+
+  useEffect(() => {
+    if (navRef.current) {
+      const height = navRef.current.clientHeight
+      setNavRefHeight(height)
+    } // eslint-disable-line react-hooks/exhaustive-deps
+  })
```

Comment:
I led you astray a little bit here. We can gain some minor efficiency adding a dependency to this useEffect hook. We can't use an empty `[]` as that will only run on the first render, but we should be able to use`[navRef.current?.clientHeight]` so that this hook only gets called when the nav height actually changes instead of at each render. You can do the same thing below with the useEffect hook that set the `formControlHeight.